test/unittests/SettingsTest.php

```diff
@@ -162,9 +163,23 @@ public function testGetBaseURL()
         $this->_configMock->expects($this->any())
             ->method('getSetting')
             ->with($this->equalTo('www'))
-            ->willReturn(array('url' => 'test.loris.ca'));
+            ->willReturn(array('url' => 'test1.loris.ca'));
 
-        $this->assertEquals("test.loris.ca", $this->_settings->getBaseURL());
+        $this->assertEquals("test1.loris.ca", $this->_settings->getBaseURL());
+    }
+
+    /**
+     * Test that getBaseURL() returns the correct value from the 
+     * 'LORIS_BASEURL' environment variable if it is set
+     *
+     * @return void
+     * @covers Settings::getBaseURL
+     */
+    public function testGetBaseURLWithLorisBaseURL()
+    {
+        //putenv sets the environment variable
+        putenv("LORIS_BASEURL=test2.loris.ca");
+        $this->assertEquals("test2.loris.ca", $this->_settings->getBaseURL());
     }
```

Comment:
after assertEquals , add this [ putenv("LORIS_BASEURL")  ] // unset this environment variable